specification/basic_concepts/basic_concepts.tex

```diff
@@ -29,38 +29,30 @@ \chapter{Basic concepts}\label{sec:basic_concepts}
 A DSDL description can be used to automatically generate the serialization/deserialization code
 for every defined data structure in a particular programming language.
 DSDL ensures that the worst case memory footprint and computational complexity per data type
-are constant and easily predictable, which is paramount for hard real-time and safety-critical applications.
+are constant and easily predictable.
```

Comment:
I think the removed statement was beneficial for emphasizing that the protocol is specifically designed for these kinds of applications. Would you not agree?

I understand that the text might be redundant in some aspects, but that is probably good for clarity.

Reply:
This is a normative section so I've tried to remove anything that might be construed as "marketing". We should move all suggestions for suitability or other text promoting the standard's use to the non-normative "introduction" section. 

That aside, this text really is more a snippet from a embedded systems course and doesn't add information about how to implement UAVCAN. It's correct but superfluous.